python/sdk/test/conftest.py

```diff
@@ -145,6 +145,13 @@ def requests():
 
     return req
 
+@pytest.fixture
+def gpu_config():
+    gpu_request = "1"
+    gpu_name = "nvidia-tesla-p4"
```

Comment:
Can you update it to try to read from environment variables first? Similar like other fixtures.

Reply:
This allows the fixture to be more configurable.